README.md

```diff
@@ -390,6 +392,19 @@ const { winc, actualTokenAmount, equivalentWincTokenAmount } =
   });
 ```
 
+#### `getTokenPriceForBytes({ bytes })`
+
+Get the current price from the Turbo Payment Service, denominated in the specified token, for uploading a specified number of bytes to Turbo.
+
+```typescript
+const turbo = TurboFactory.unauthenticated({ token: 'solana' });
+const { tokenPrice } = await turbo.getTokenPriceForBytes({
+  bytes: 1024 * 1024 * 100,
```

Comment:
Could this be `byteCount` rather than `bytes`? `bytes` could be interpreted as actual raw bytes like a buffer or something.

Reply:
1db8ece..171eadd